pkgs/development/python-modules/meliae/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, cython
+}:
+
+buildPythonPackage rec {
+  pname = "meliae";
+  version = "0.4.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16gcgjzcjlbfarwlj18fn089bz34kzfw8varbzxsi9ma0amijrcp";
+  };
+
+  checkInputs = [ cython ];
```

Comment:
Not needed if it is already in `propagatedBuildInputs`

Reply:
Fixed, thanks.